docker-compose.yml

```diff
@@ -42,19 +42,38 @@ services:
     security_opt:
       - label=disable
 
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
+    container_name: zookeeper
+    env_file:
+      - ./conf/zookeeper.env
+    ports:
+      - 2181:2181
+
+  kafka:
```

Comment:
Couldn't we run an image with contains both zookeper and kafka ?

Reply:
Yes, but we would have to build it ourself. This is standard way according to kafka docs.